packaging/generate_tarball.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+NAME=$1
+DUNE_PREFIX="$PREFIX"/"$SUBPREFIX"
+mkdir -p "$DUNE_PREFIX"/scripts
+mkdir -p "$DUNE_PREFIX"/src
+mkdir -p "$DUNE_PREFIX"/licenses
+
+#echo ""$PREFIX" ** "$SUBPREFIX" ** "$DUNE_PREFIX""
+
+# install
+cp -R "$BUILD_DIR"/dune* "$DUNE_PREFIX" || exit 1
+cp -R "$BUILD_DIR"/Dockerfile* "$DUNE_PREFIX" || exit 1
+cp -R "$BUILD_DIR"/bootstrap* "$DUNE_PREFIX" || exit 1
+cp -R "$BUILD_DIR"/src/* "$DUNE_PREFIX"/src || exit 1
+cp -R "$BUILD_DIR"/scripts/* "$DUNE_PREFIX"/scripts || exit 1
+cp -R "$BUILD_DIR"/LICENSE* "$DUNE_PREFIX"/licenses || exit 1
+
```

Comment:
Add the following so users don't need to set their own path to use dune:
```bash
# Add symbolic link
mkdir ./usr/bin
ln -s /"$DUNE_PREFIX/dune" ./usr/bin/dune
```

Reply:
https://github.com/AntelopeIO/DUNE/pull/75#discussion_r1053277121